### PR TITLE
Production Deploy

### PR DESCRIPTION
**Pull request by tcmacdonald**

### CODE REVIEW

File: src/components/crds-close-button/crds-close-button.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.circle {
+  background-color: $cr-cyan;
```
Comment (kellgauthier) [inline src/components/crds-close-button/crds-close-button.scss:2]: Old brand color - does this need to be changed?

File: src/components/form/fred/fred.scss [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+.input-group .form-control, .crds-styles .input-group .form-control {
+  margin: 0px;
+  background-color: #eeeeee;
+}
+
+.input-group-addon, .crds-styles .input-group-addon {
+  background-color: #eeeeee;
+  border: 1px solid;
+  border-color: #d7d7d7;
```
Comment (j-brammer) [inline src/components/form/fred/fred.scss:51]: If we are not going to use a variable here, can we get a comment as to why it's hard coded for future code travelers?

File: src/shared/shared.scss [hunk 3]
```diff
@@ -80,26 +82,27 @@ hr {
...
 
 .text-cyan {
-  color: #0095D9 !important;
+  color: $cr-cyan !important;
 }
+
 .text-cyan-light {
-  color: #8BCEED !important;
+  color: $cr-cyan-light !important;
 }
```
Comment (j-brammer) [inline src/shared/shared.scss:102]: shouldn't this be gone?

File: src/shared/shared.scss [hunk 4]
```diff
@@ -322,32 +324,32 @@ hr {
...
   }
 
   &.btn-cyan {
-    background-color: #0095d9;
-    border-color: #0095d9;
-    color: #fff;
+    background-color: $cr-cyan;
+    border-color: $cr-cyan;
+    color: $cr-white;
   }
```
Comment (j-brammer) [inline src/shared/shared.scss:343]: more cyan

**Pull request merged**